{
    "type": "bundle",
    "id": "bundle--994eb26b-6135-452e-bfd0-34d75a637766",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--70561566-76d4-4b1c-932e-c14343de953d",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2022-05-18T00:40:03.038204Z",
            "modified": "2022-05-18T00:40:03.038204Z",
            "name": "CVE-2021-38872",
            "description": "IBM DataPower Gateway 10.0.2.0, 10.0.3.0, 10.0.1.0 through 10.0.1.4, and 2018.4.1.0 through 2018.4.1.17 could allow a remote user to cause a denial of service by consuming resources with multiple requests. IBM X-Force ID: 208348.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2021-38872"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}